Test-network.md

```diff
@@ -60,6 +60,13 @@ At the moment it's assinging ports in the following way:
 * Satellite Console starts on port `10100`
 * Storage Nodes start from port `11000`
 
+To get access to a gateway and test your keys, you open http://127.0.0.1/9000 in a web browser.
```

Comment:
It's a good moment to update also the "Storage Nodes start from port `11000`" line above.

Now we have private(internal) and public(external) ports for storage nodes. Public ports starts from 12000, private - from 13000. There are no 1100X ports anymore.